pkg/db/client.go

```diff
@@ -146,6 +169,54 @@ func (c *ClientSqlx) Exec(ctx context.Context, query string, args ...interface{}
 	return c.db.ExecContext(ctx, query, args...)
 }
 
+func (c *ClientSqlx) ExecMultiInTx(ctx context.Context, sqllers ...Sqler) (results []sql.Result, err error) {
```

Comment:
In the interface you have called in sqlers, so i assume it should be the same on implementations.

```suggestion
func (c *ClientSqlx) ExecMultiInTx(ctx context.Context, sqlers ...Sqler) (results []sql.Result, err error) {
```

Reply:
:+1: